core/block_svg.js

```diff
@@ -965,6 +977,12 @@ Blockly.BlockSvg.prototype.onMouseMove_ = function(e) {
       }
       this.setDragging_(true);
       this.moveToDragSurface_();
+
+
+      if (!Blockly.draggedConnectionManager_) {
+        Blockly.draggedConnectionManager_ =
```

Comment:
so there's one draggedConnectionManager per blockly instance? I want to make sure we test this fairly thoroughly with multi-workspaces and mouse ups/mouse moves/mousedowns across multiple sets of workspaces and ones that occur outside of the browser window/frame.